hws/src/dr_matcher.py

```diff
@@ -125,15 +125,24 @@ def dump_str(self, verbosity):
     def dump_matcher_statistcs(self):
         row_log_sz = self.attr.get_row_log_sz()
         col_log_sz = self.attr.get_col_log_sz()
+        tbl_type = _db._tbl_type_db.get(self.data.get("tbl_id"))
         _str = "Statistics: distribution: "
-        if self.match_ste_1_id != None:
+        if tbl_type == DR_TBL_TYPE_FDB_UNIFIED:
+            _str += "RX/TX: "
+        elif tbl_type == DR_TBL_TYPE_FDB:
             _str += "RX: "
+
         _str += get_fw_ste_distribution_statistics(self.match_ste_0_id, row_log_sz, col_log_sz)
+
+        if tbl_type == DR_TBL_TYPE_FDB_UNIFIED:
+            return _str + "\n"
+        elif tbl_type == DR_TBL_TYPE_FDB:
```

Comment:
Why do you need this 'if' here?
If it was nic_rtx/tx, then you have only one match ste.
If it was unified, you're out.
You can put back the "TX:" to where it was before.